styles/widgets/generic/validation.generic.less

```diff
@@ -1,3 +1,4 @@
+@import (once) "./overlay.generic.less";
 @import (once) "../base/validation.less";
```

Comment:
Maybe it's better to strengthen CSS selectors in base/validation.less and import theme-specific less after the base one?

Reply:
I'll change the order